.github/workflows/cron.yml

```diff
@@ -46,9 +46,17 @@ jobs:
           username: ${{ env.GH_USER }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      # `oras push` fails if repository contains uppercase characters
+      # required for testing in forks
+      - name: GH repo to lower case
+        id: github-repo-lower-case
+        uses: ASzc/change-string-case-action@v6
+        with:
+          string: ${{ github.repository }}
+
```

Comment:
This is not a required change. But it will help me (and other contributors with capital letters in their names) work with the changes from the fork.

Reply:
What if using bash? To reduce the risk of supply chain attacks, I want to avoid introducing third-party actions unless it's truly necessary.

```
      lowercase_repo=$(echo "${{ github.repository }}" | tr '[:upper:]' '[:lower:]')
```

Reply:
no problem. changed in lowercase_repo=$(echo "${{ github.repository }}" | tr '[:upper:]' '[:lower:]')

Reply:
Now, we don't need this step and can merge it into the same step.

```
      - name: Upload assets to GHCR
        run: |
          lowercase_repo=$(echo "${{ github.repository }}" | tr '[:upper:]' '[:lower:]')"
          oras version
          oras push --artifact-type application/vnd.aquasec.trivy.config.v1+json \
          ghcr.io/${lowercase_repo}:${DB_VERSION} \
          javadb.tar.gz:application/vnd.aquasec.trivy.javadb.layer.v1.tar+gzip